fineract-provider/src/main/java/org/apache/fineract/portfolio/self/loanaccount/api/SelfLoansApiResourceSwagger.java

```diff
@@ -165,9 +165,9 @@ private GetLoansLoanIdInterestCalculationPeriodType() {}
             public String description;
         }
 
-        static final class GetLoansLoanIdTimeline {
+        static final class GetSelfLoanIdTimeline {
```

Comment:
This was intentional. the class with ```GetLoansLoanIdTimeline``` is already existing and they were overriding each other.